luna/gateware/usb/usb2/descriptor.py

```diff
@@ -344,7 +344,7 @@ def generate_rom_content(self):
         # Chunk our ROM into a collection of entries...
         rom_entries = (rom[(element_size * i):(element_size * i) + element_size] for i in range(total_elements))
 
-        # ... and then convert that into an initializer value in the format nMigen ROMs like (integers).
+        # ... and then convert that into an initializer value in the format Amaranth HDL ROMs like (integers).
```

Comment:
This one is more marginal but I would go with `Amaranth ROMs`.

I suppose the rule I'm trying to suggest is that within compound nouns, it's confusing to use the full "Amaranth HDL" name and clearer to just go with "Amaranth".